### DISCUSSION TITLE
My domain is not connected to my account anymore!

### DISCUSSION THREAD

**Discussion by jakobpack** [vercel/community#7008] [Account Issue] [answered]
### Question

I had one domain transferred in into Vercel and the configuration is still working. I used a different DNS provider at some point. Now I wanted to reconfigure the domain, but it's not listed anymore in my account...

**Comment by amyegan** [comment, marked as answer]
Hey @jakobpack. This may be related to the switch from personal account deployments to [free hobby teams](https://github.com/orgs/vercel/discussions/6298). If you don't see the domain on the team's _Domains_ tab, you should check your personal account's _domains_ list at [vercel.com/account/domains](https://vercel.com/account/domains). From there, you can [move the domain to your new team](https://vercel.com/docs/projects/domains/transfer-your-domain) if you prefer to manage it from the team dashboard.

Please give that a try and let me know if you run into any problems 🙏

**Reply by jakobpack** [reply]
Oh thank god! There it is. Thank you!
